src/locales/cy/translation.json

```diff
@@ -328,8 +327,7 @@
         "subHeading4": "Os ydych angen help",
         "paragraph8": "Cysylltwch â ni os:",
         "bullet3": "ydych angen help i orffen profi eich hunaniaeth gyda Swyddfa’r Post",
-        "bullet4": "hoffech beidio â mynd i Swyddfa’r Post ac eisiau profi eich hunaniaeth mewn ffordd arall",
-        "contactLinkHrefF2fCustom": "https://signin.account.gov.uk/contact-us-questions?theme=proving_identity&referer="
```

Comment:
This isn't the same in English and Welsh but it should be, so it doesn't need to be in the translation file